samples/sensor/bme280/app.overlay

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * By default, this overlay sets up a single BME280 sensor
+ * on the standard Arduino header I2C pins. Adjust the aliases
+ * definition below to switch to SPI.
+ *
+ * If your board's devicetree has "arduino_i2c" and "arduino_spi" node
+ * labels, you can use this overlay as-is, wiring the BME280 sensor to
+ * the breakout header of your choice accordingly:
+ *
+ * https://upload.wikimedia.org/wikipedia/commons/c/c9/Pinout_of_ARDUINO_Board_and_ATMega328PU.svg
+ *
+ * If you're not sure about your board, just compile the application and see.
+ * You'll get a build error if there is no "arduino_i2c" or "arduino_spi".
+ *
+ * If your board does not have an Arduino header, you can create a file
+ * named boards/YOUR_BOARD_NAME.overlay in this sample and set up the
+ * sensor device on an I2C or SPI bus available in your devicetree.
+ * See boards/adafruit_feather_m0_basic_proto.overlay for an example.
+ */
+
+/ {
+	aliases {
```

Comment:
What about `chosen` ?

Reply:
> What about `chosen` ?

We generally use aliases for samples, and chosen for systemwide settings.

Reply:
ok